src/mito2/src/wal/raw_entry_reader.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use futures::stream::BoxStream;
+use store_api::logstore::entry::RawEntry;
+use store_api::storage::RegionId;
+
+use crate::error::Result;
+use crate::wal::EntryId;
+
+/// A stream that yields [RawEntry].
+pub type RawEntryStream<'a> = BoxStream<'a, Result<RawEntry>>;
+
+/// The context of [RawEntryReader].
+pub(crate) enum LogStoreReadCtx {
```

Comment:
Maybe rename it to namespace and remove those existing namespace design.

Reply:
I renamed it to `LogStoreNamespace`. Maybe remove the namespace in the next refactoring.